.github/workflows/test.yml

```diff
@@ -0,0 +1,102 @@
+name: Test
+
+on:
+  push:
+    branches:
+      - master
+
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  lint-hbs:
+    name: Lint Handlebars
+    runs-on: [ubuntu-latest]
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v1
+
+    - name: Lint Handlebars
+      uses: Borales/actions-yarn@master
```

Comment:
`borales/actions-yarn` is deprecated. Use `actions/setup-node`

Reply:
Also, how can you lint withou installing the dependencies?

Reply:
Changed